tests/core/test_model.py

```diff
@@ -2582,6 +2582,150 @@ def my_model(context, **kwargs):
         )
 
 
+def test_model_physical_properties_labels() -> None:
+    sql_model = load_sql_based_model(
+        d.parse(
+            """
+        MODEL (
+            name test_schema.test_model,
+            physical_properties [
+                labels = [('test-label', 'label-value')]
+            ]
+        );
+        SELECT a FROM tbl;
+        """
+        )
+    )
+    assert sql_model.physical_properties == {
+        "labels": exp.Array(
```

Comment:
let's use the builder api for the tests,

exp.array("('test-label', 'label-value')")

Reply:
Right, it's more compact 👍